packages/create-spectacle/src/cli.ts

```diff
@@ -72,14 +74,43 @@ const main = async () => {
               return val.trim().length > 0 ? true : 'Name is required';
             }
           },
+          /**
+           * Type of deck.
+           * Needs to be before overwrite confirmation so we can determine if folder/file already exists.
+           */
+          {
+            type: 'select',
+            name: ArgName.type as string,
+            message: 'What type of deck do you want to create?',
+            choices: DeckTypeOptions,
+            initial: (() => {
+              const ind = DeckTypeOptions.findIndex((o) => o.value === type);
+              return ind > -1 ? ind : 0;
+            })()
+          },
           // If output directory already exists, prompt to overwrite
           {
-            type: (val) => (isFolderNameAvailable(val) ? null : 'confirm'),
+            type: (_, answers) =>
+              isOutputPathAvailable(
+                answers?.[ArgName.name],
+                answers?.[ArgName.type] === 'onepage'
+              )
+                ? null
+                : 'confirm',
             name: ArgName.overwrite as string,
-            message: (val) =>
-              `Target directory ${formatProjectDirName(
-                val
-              )} already exists. Overwrite and continue?`
+            message: (_, answers) => {
+              const name = answers?.[ArgName.name];
+              const type = answers?.[ArgName.type];
+              if (type === 'onepage') {
+                return `File ${formatProjectOutputPath(
+                  name
+                )}.html already exists. Overwrite it?`;
+              } else {
+                return `Target directory ${formatProjectOutputPath(
+                  name
+                )} already exists. Overwrite and continue?`;
```

Comment:
This is really nit picky but we have two different prompt strings for overwriting. Let's standardize on "Override and continue?" for line 107 as well.

Reply:
Ooooh, detailed eye! Nice!